source/StigData/Processed/WindowsServer-2016-DC-2.3.xml

```diff
@@ -3648,6 +3655,7 @@ Administrators - Full Control - This key and subkeys
 SYSTEM - Full Control - This key and subkeys
 CREATOR OWNER - Full Control - Subkeys only
 ALL APPLICATION PACKAGES - Read - This key and subkeys
+Server Operators - Read - This Key and subkeys
```

Comment:
There should be one more rule 224835.d - Server Operators - Read - This Key and subkeys

Reply:
Is this a Hardcoded rule that needs to be added?